CKI-project/tests-beaker
# Remove RhtsRequires from filesystems/nfs/connectathon

1. Remove "RhtsRequires: test(/kernel/include)" from Makefile
2. Add kstub.sh which is the copy of runtest.sh of package kernel-kernel-include then remove kvercmp.sh because function kvercmp() can be replaced by K_Vercmp()
3. Remove default server list and move it to kpet-db (MR#693)

A beaker job is done to verify the fix, please refer to job `4187211`.
